python/ray/state.py

```diff
@@ -382,8 +382,12 @@ def client_table(self):
             Information about the Ray clients in the cluster.
         """
         self._check_connected()
+        client_table = _parse_client_table(self.redis_client)
 
-        return _parse_client_table(self.redis_client)
+        for client in client_table:
+            client["alive"] = (client["EntryType"] !=
+                               gcs_utils.ClientTableData.DELETION)
```

Comment:
Are there possibly other `EntryType`s that should be considered as not-alive?

Reply:
I think this is it - I'm using this as a reference (ray/state.py:cluster_resources):

```python
    def cluster_resources(self):
        """Get the current total cluster resources.

        Note that this information can grow stale as nodes are added to or
        removed from the cluster.

        Returns:
            A dictionary mapping resource name to the total quantity of that
                resource in the cluster.
        """
        self._check_connected()

        resources = defaultdict(int)
        clients = self.client_table()
        for client in clients:
            # Only count resources from latest entries of live clients.
            if client["EntryType"] != gcs_utils.ClientTableData.DELETION:
                for key, value in client["Resources"].items():
                    resources[key] += value
        return dict(resources)
```